flymake-languagetool.el

```diff
@@ -214,30 +223,38 @@ non-nil."
 (when flymake-languagetool-use-categories
   (flymake-languagetool--category-setup))
 
+;; Ignore some faces
+(defun flymake-languagetool-ignore-at-pos-p (pos faces-to-ignore)
+  (let ((x (get-text-property pos 'face)))
+    (if (listp x)
+        (cl-intersection faces-to-ignore x)
+      (memq x faces-to-ignore))))
```

Comment:
Let's change this function's prefix with two dashes (`flymake-languagetool--ignore-at-pos-p`) since it's for internal use. And please add a documentation for this function even it's very simple and straight forward. ;)

Reply:
Ok, will get this done

Reply:
Also, `ensure-list` can be used to simplify.